src/navigations/HomeBottomTabs.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-use-before-define */
 import * as React from 'react';
-import {SafeAreaView, StyleSheet, View} from 'react-native';
+import {SafeAreaView, StyleSheet, Text, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Text` has been added. This is likely to be used in the component for displaying text elements.
```diff
+ import {SafeAreaView, StyleSheet, Text, View} from 'react-native';
- import {SafeAreaView, StyleSheet, View} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->